codegen/method.go

```diff
@@ -431,10 +443,89 @@ func findParamsAnnotation(
 	walkFieldGroups(fields, visitor)
 
 	if identifier == "" {
-		return "", false
+		return "", required, false
```

Comment:
nit: more than one return of the same type may want to label all of them, i.e.   (required, ok bool)